### PR TITLE
add NX_COUNT

### LINKED ISSUE #932: What unit type to use for "counts"?
NeXus does not have a unit type that matches `counts`.  (number of photons, neutrons, or other counting events)

* What do people use?
* Should we add `NX_COUNTS`?

### PR DESCRIPTION
**Pull request by prjemian**
- FIX #932
- REF #943

### CODE REVIEW

File: nxdlTypes.xsd [hunk 1]
```diff
@@ -113,6 +113,16 @@
 		<xs:restriction base="xs:string" />
 	</xs:simpleType>
 	
+	<xs:simpleType name="NX_COUNT">
+		<xs:annotation>
+			<xs:documentation>
+				units of quantity of item(s) such as number of photons,
+				neutrons, or other counting events
```
Comment (PeterC-DLS) [inline nxdlTypes.xsd:120]: Could add pulses

File: nxdlTypes.xsd [hunk 2]
```diff
@@ -113,6 +113,16 @@
...
+			</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="nxdl:NX_INT" />
```
Comment (PeterC-DLS) [inline nxdlTypes.xsd:123]: Not string as all units attributes are strings?

File: nxdlTypes.xsd (hunk 1 above)
Comment (prjemian) [reply nxdlTypes.xsd:120]: added

File: nxdlTypes.xsd (hunk 2 above)
Comment (prjemian) [reply nxdlTypes.xsd:123]: Right!  What about these?  Same correction?

https://github.com/nexusformat/definitions/blob/99fb0e0ee50e57ed049566ff16f3159a083ddf3c/nxdlTypes.xsd#L259-L268

https://github.com/nexusformat/definitions/blob/99fb0e0ee50e57ed049566ff16f3159a083ddf3c/nxdlTypes.xsd#L290-L297

https://github.com/nexusformat/definitions/blob/99fb0e0ee50e57ed049566ff16f3159a083ddf3c/nxdlTypes.xsd#L343-L352

**Review by PeterC-DLS — APPROVED** [review]

File: nxdlTypes.xsd (hunk 2 above)
Comment (prjemian) [reply nxdlTypes.xsd]: created as new issue

**Pull request merged**